plugin/phpactor.vim

```diff
@@ -70,9 +79,18 @@ function! phpactor#Complete(findstart, base)
     let issues = result['issues']
 
     let completions = []
+    let g:_phpactorCompletionMeta = {}
+
     if !empty(suggestions)
         for suggestion in suggestions
-            call add(completions, { 'word': suggestion['name'], 'menu': suggestion['info'], 'kind': suggestion['type']})
+            let completion = { 
+                        \ 'word': suggestion['name'], 
+                        \ 'menu': suggestion['short_description'],
+                        \ 'kind': suggestion['type'],
+                        \ 'dup': 1
```

Comment:
Could add this as a bug fix - it allows duplicate name suggestions (e.g. Symfony `Request` and Phpactor `Request`)